encodings/sequence/src/compute/compare.rs

```diff
@@ -26,23 +25,15 @@ impl CompareKernel for SequenceVTable {
         };
 
         // Check if there exists an integer solution to const = base + (0..len) * multiplier.
-        let set_idx: Option<usize> = match_each_integer_ptype!(lhs.ptype(), |P| {
-            let c = constant
+        let set_idx: Option<usize> = find_intersection_scalar(
```

Comment:
you don't need the annotaion here anymore since this is an explicit function call